shared/utils/src/test/roundvalue.unit.spec.js

```diff
@@ -1,5 +1,5 @@
 import tape from 'tape'
-import { roundValueAuto } from '../roundValue.js'
+import { roundValueAuto, roundValue2 } from '../roundValue.js'
```

Comment:
decimalPlacesUntilFirstNonZero is a key function. why is it not tested?

Reply:
Looks like it wasn't considered when this file was created.

Reply:
will leave it as good first issue